pkg/bpf/setup_linux.go

```diff
@@ -75,5 +76,19 @@ func Setup(l *zap.Logger) error {
 		return errors.Wrap(err, "failed to initialize filter map")
 	}
 	l.Info("Filter map initialized successfully", zap.String("path", plugincommon.MapPath), zap.String("Map name", plugincommon.FilterMapName))
+
+	// Delete existing conntrack map file.
+	err = os.Remove(plugincommon.MapPath + "/" + plugincommon.ConntrackMapName)
```

Comment:
This attempts to remove the map file regardless of if its present or not then handles the error if it's a specific type or not. The code above does the same, does it make sense to check if the file exists and then delete it? Perhaps a reason exists for this approach?

Reply:
we can definitely do an `os.Stat` first, but then we will have to handle the err returned from it, wondering if there is a cleaner way to do this in Go 🤔

Reply:
Can this map not exist in any scenario? If so, then let's just handle two errors separately: failure to remove existing map, and failure to delete non-existent object. Then it doesn't matter whether we check for existence first or not, and can do either way

Reply:
these maps will not exist when Retina is installed for the first time